swag_exporter/setup_info.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "swag_export",
+  "version": "0.0.1",
+  "url": "https://voereir.com",
+  "author": "VoerEir AB",
+  "author_email": "info@voereir.com",
+  "description": "Swag Exporter for exporting swagger spec json as html to pdf"
```

Comment:
*JSON
*HTML
*PDF

Add a full stop at the end.

Reply:
done!!

Reply:
Nope. They should be capitalized.